src/contracts/SavingCircles.sol

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+import {OwnableUpgradeable} from '@openzeppelin-upgradeable/access/OwnableUpgradeable.sol';
+import {IERC20} from '@openzeppelin/token/ERC20/IERC20.sol';
+import {ReentrancyGuard} from '@openzeppelin/utils/ReentrancyGuard.sol';
+
+import {ISavingCircles} from '../interfaces/ISavingCircles.sol';
+
+/**
+ * @title Saving Circles
+ * @notice Simple implementation of a rotating savings and credit association (ROSCA) for ERC20 tokens
+ * @author Breadchain Collective
+ * @author @RonTuretzky
+ * @author @bagelface
```

Comment:
on the YD contract everyone has their ENS, wdyt our convention should be for tagging authors?

Reply:
ya i personally like ENS, i think we should just put it however the author requests. do you want me to use your ENS for this?

Reply:
if you let me know which ENS to use for you i will change. i changed mine